include/sys/zio_priority.h

```diff
@@ -29,8 +29,9 @@ typedef enum zio_priority {
 	ZIO_PRIORITY_ASYNC_WRITE,	/* spa_sync() */
 	ZIO_PRIORITY_SCRUB,		/* asynchronous scrub/resilver reads */
 	ZIO_PRIORITY_NUM_QUEUEABLE,
-
-	ZIO_PRIORITY_NOW		/* non-queued i/os (e.g. free) */
+	ZIO_PRIORITY_NOW,		/* non-queued i/os (e.g. free) */
+	ZIO_PRIORITY_COUNT,		/* Count of all priorities. Leave */
```

Comment:
There's no need to add this, the patch should be using `ZIO_PRIORITY_NUM_QUEUEABLE` throughout (even if it is a terrible name).  There exactly this many `vdev_queue_class_t`'s defined in a `vdev_queue_t`.